package.json

```diff
@@ -2,52 +2,31 @@
   "name": "@stellar/anchor-tests",
   "version": "0.1.0",
   "description": "stellar-anchor-tests is a library and command line interface for testing Stellar anchors.",
-  "main": "./lib/index.js",
-  "types": "./lib/index.d.ts",
-  "files": [
-    "./lib"
+  "private": true,
+  "workspaces": [
+    "@stellar/anchor-tests",
+    "app/ui",
+    "app/server"
   ],
-  "bin": {
-    "stellar-anchor-tests": "lib/cli.js"
-  },
   "repository": {
     "type": "git",
     "url": "https://github.com/stellar/stellar-anchor-tests.git"
   },
   "license": "Apache-2.0",
   "scripts": {
-    "build": "tsc",
-    "build:docs": "typedoc",
+    "build:anchor-tests": "yarn workspace @stellar/anchor-tests build",
+    "build:docs": "yarn workspace @stellar/anchor-tests build:docs",
     "prepare": "husky install"
   },
   "homepage": "https://github.com/stellar/stellar-anchor-tests",
   "author": "Stellar Development Foundation <hello@stellar.org>",
   "bugs": {
     "url": "https://github.com/stellar/stellar-anchor-tests/issues"
   },
-  "prettier": "@stellar/prettier-config",
```

Comment:
I only kept husky deps in the root. Yarn should automatically de-dupe modules, so no need to worry about repeated dependencies in subrepos